src/plugins/notebook/components/Notebook.vue

```diff
@@ -1,16 +1,38 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2021, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
```

Comment:
I think this is only for .js files.

Reply:
I've seen it in other vue files, but it's possible I put it there :) prolly a better q for @akhenry